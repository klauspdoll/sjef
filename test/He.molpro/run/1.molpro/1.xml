<?xml version="1.0"?>
<molpro xmlns="http://www.molpro.net/schema/molpro-output"
  xmlns:xsd="http://www.w3.org/1999/XMLSchema"
  xmlns:cml="http://www.xml-cml.org/schema"
  xmlns:stm="http://www.xml-cml.org/schema"
  xmlns:xhtml="http://www.w3.org/1999/xhtml">
 <job>
  <stm:metadataList>
   <stm:metadata name="dc:date" content="2022-11-10T18:31:53+00:00"/>
   <stm:metadata name="dc:creator" content="peterk"/>
   <stm:metadata name="cmlm:insilico" content="Molpro"/>
  </stm:metadataList>
  <platform pid="64363">
   <version major="2022" minor="3" SHA="fa99d9912ba884475db19361bf6c586b05586219"
     integer_bits="64" parallelism="serial">
    2022.3
    <date year="2022" month="11" day="10" hour="18" minute="31" second="53">
     2022-11-10T18:31:53
    </date>
   </version>
   <licence id="peterk"/>
   <parallel processes="1" nodes="1" all_processes="1" openmp="0"/>
   <dimensions natom="400" nvalence="500" nvalence_pno="1000" nbasis="12000" nstate="100"
     nsymm="16" nrec="512" nprim="2" ncol="100"/>
  </platform>
  <jobstep command="RHF-SCF" commandset="SCFPRO">
   <cml:molecule>
    <cml:symmetry pointGroup="D2h">
     <cml:transform3 title="generator" id="X">
      -1  0  0  0  0  1  0  0  0  0  1  0  0  0  0  1
     </cml:transform3>
     <cml:transform3 title="generator" id="Y">
       1  0  0  0  0 -1  0  0  0  0  1  0  0  0  0  1
     </cml:transform3>
     <cml:transform3 title="generator" id="Z">
       1  0  0  0  0  1  0  0  0  0 -1  0  0  0  0  1
     </cml:transform3>
    </cml:symmetry>
    <cml:atomArray>
     <cml:atom id="a1" elementType="He" x3="0.0" y3="0.0" z3="0.0"/>
    </cml:atomArray>
    <cml:bondArray>
    </cml:bondArray>
   </cml:molecule>
   <property name="Energy" method="RHF" principal="true" stateSymmetry="1" stateNumber="1"
     value="-2.85516047724274"/>
   <property name="One-electron energy" method="RHF" value="-3.88202510260425"/>
   <property name="Two-electron energy" method="RHF" value="1.02686462536151"/>
   <property name="Kinetic energy" method="RHF" value="2.85517613807823"/>
   <property name="Nuclear energy" method="RHF" value="0.0"/>
   <property name="Virial quotient" method="RHF" value="-0.999994514931921"/>
   <property name="Dipole moment" method="RHF" principal="true" stateSymmetry="1"
     stateNumber="1" value="0.0 0.0 0.0"/>
   <time start="18:31:53" end="18:31:53" cpu="0.14" system="0.04" real="0.2"/>
   <storage units="megabyte" memory="244.14" sf="0.0" df="33.05" eaf="0.0"/>
   <summary overall_method="RHF/cc-pVDZ"/>
  </jobstep>
  <jobstep command="MP2" commandset="CCSD">
   <property name="energy" method="Reference" stateSymmetry="1" stateNumber="1"
     value="-2.85516047724274"/>
   <single orbital="1.1" energy="0.0"/>
   <single orbital="-1.1" energy="0.0"/>
   <pair orbital1="1.1" orbital2="-1.1" energy="-0.01291417"/>
   <pair orbital1="-1.1" orbital2="1.1" energy="-0.01291417"/>
   <property name="singlet pair energy" method="MP2" stateSymmetry="1" stateNumber="1"
     value="-0.258283395513885E-01"/>
   <property name="triplet pair energy" method="MP2" stateSymmetry="1" stateNumber="1"
     value="0.0"/>
   <property name="correlation energy" method="MP2" stateSymmetry="1" stateNumber="1"
     value="-0.258283395513885E-01"/>
   <property name="total energy" method="MP2" principal="true" stateSymmetry="1"
     stateNumber="1" value="-2.88098881679413"/>
   <time start="18:31:53" end="18:31:53" cpu="0.02" system="0.0" real="0.02"/>
   <storage units="megabyte" memory="243.89" sf="0.0" df="33.05" eaf="0.0"/>
   <summary overall_method="MP2/cc-pVDZ"/>
  </jobstep>
  <jobstep command="CCSD" commandset="CCSD">
   <property name="energy" method="Reference" stateSymmetry="1" stateNumber="1"
     value="-2.85516047724274"/>
   <single orbital="1.1" energy="0.0"/>
   <single orbital="-1.1" energy="0.0"/>
   <pair orbital1="1.1" orbital2="-1.1" energy="-0.01621718"/>
   <pair orbital1="-1.1" orbital2="1.1" energy="-0.01621718"/>
   <property name="singlet pair energy" method="CCSD" stateSymmetry="1" stateNumber="1"
     value="-0.324343539760497E-01"/>
   <property name="triplet pair energy" method="CCSD" stateSymmetry="1" stateNumber="1"
     value="0.0"/>
   <property name="correlation energy" method="CCSD" stateSymmetry="1" stateNumber="1"
     value="-0.324343539760411E-01"/>
   <property name="total energy" method="CCSD" principal="true" stateSymmetry="1"
     stateNumber="1" value="-2.88759483121878"/>
   <time start="18:31:53" end="18:31:53" cpu="0.02" system="0.0" real="0.03"/>
   <storage units="megabyte" memory="243.89" sf="0.0" df="33.05" eaf="0.0"/>
   <summary overall_method="CCSD/cc-pVDZ"/>
  </jobstep>
  <input>
   <p>geometry={He}</p>
   <p>rhf</p>
   <p>mp2</p>
   <p>ccsd</p>
  </input>
  <diagnostics warnings="0"/>
 </job>
</molpro>
