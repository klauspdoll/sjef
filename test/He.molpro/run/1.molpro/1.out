
 Working directory              : /var/folders/qd/f26v1rbd3g5341rsg3s9j4v80000gn/T//molpro.gZlcGQXhSe/
 Global scratch directory       : /var/folders/qd/f26v1rbd3g5341rsg3s9j4v80000gn/T//molpro.gZlcGQXhSe/
 Wavefunction directory         : /Users/peterk/wfu/
 Main file repository           : /var/folders/qd/f26v1rbd3g5341rsg3s9j4v80000gn/T//molpro.gZlcGQXhSe/

 id        : peterk

 Nodes          nprocs
 pjk2022.connect    1
 GA implementation: MPI file
 GA implementation (serial work in mppx): MPI file

 Using customized tuning parameters: mindgm=1; mindgv=20; mindgc=4; mindgr=1; noblas=0; minvec=7
 default implementation of scratch files=df  


 Variables initialized (1022), CPU time= 0.00 sec
 geometry={He}
 rhf
 mp2
 
 
 ccsd
 
 Commands initialized (841), CPU time= 0.01 sec, 683 directives.
 Default parameters read. Elapsed time= 0.05 sec

 Checking input...
 Passed
1


                                         ***  PROGRAM SYSTEM MOLPRO  ***
                                       Copyright, TTI GmbH Stuttgart, 2015
                                    Version 2022.3 linked Thu Nov 10 18:04:24 2022


 **********************************************************************************************************************************
 LABEL *                                                                                 
  64 bit serial version                                                                  DATE: 10-Nov-22          TIME: 18:31:53  
 **********************************************************************************************************************************

 SHA1:             fa99d9912ba884475db19361bf6c586b05586219
 **********************************************************************************************************************************

 Memory per process:        32 MW
 Total memory per node:     32 MW

 GA preallocation disabled
 GA check disabled

 Variable memory set to 32.0 MW



 Recomputing integrals since basis changed


 Using spherical harmonics

 Library entry HE     S cc-pVDZ              selected for orbital group  1
 Library entry HE     P cc-pVDZ              selected for orbital group  1


 PROGRAM * SEWARD (Integral evaluation for generally contracted gaussian basis sets)     Author: Roland Lindh, 1990

 Geometry written to block  1 of record 700

 Orientation using atomic masses  
 Molecule type: Atom
 Symmetry elements: X,Y,Z
 Rotational constants:       0.0000000      0.0000000      0.0000000 GHz  (calculated with average atomic masses)

 Point group  D2h 



 ATOMIC COORDINATES

 NR  ATOM    CHARGE       X              Y              Z

   1  HE      2.00    0.000000000    0.000000000    0.000000000

 NUCLEAR CHARGE:                    2
 NUMBER OF PRIMITIVE AOS:           7
 NUMBER OF SYMMETRY AOS:            7
 NUMBER OF CONTRACTIONS:            5   (    2Ag  +    1B3u +    1B2u +    0B1g +    1B1u +    0B2g +    0B3g +    0Au  )
 NUMBER OF INNER CORE ORBITALS:     0   (    0Ag  +    0B3u +    0B2u +    0B1g +    0B1u +    0B2g +    0B3g +    0Au  )
 NUMBER OF OUTER CORE ORBITALS:     0   (    0Ag  +    0B3u +    0B2u +    0B1g +    0B1u +    0B2g +    0B3g +    0Au  )
 NUMBER OF VALENCE ORBITALS:        1   (    1Ag  +    0B3u +    0B2u +    0B1g +    0B1u +    0B2g +    0B3g +    0Au  )


 NUCLEAR REPULSION ENERGY    0.00000000

 Eigenvalues of metric

         1 0.111E+00 0.189E+01
         2 0.100E+01
         3 0.100E+01
         5 0.100E+01


 Contracted 2-electron integrals neglected if value below      1.0D-11
 AO integral compression algorithm  1   Integral accuracy      1.0D-11

     0.262 MB (compressed) written to integral file (100.0%)


 NUMBER OF SORTED TWO-ELECTRON INTEGRALS:         45.     BUFFER LENGTH:  32768
 NUMBER OF SEGMENTS:   1  SEGMENT LENGTH:         45      RECORD LENGTH: 524288

 Memory used in sort:       0.56 MW

 SORT1 READ          61. AND WROTE          45. INTEGRALS IN      1 RECORDS. CPU TIME:     0.00 SEC, REAL TIME:     0.00 SEC
 SORT2 READ          45. AND WROTE          45. INTEGRALS IN      1 RECORDS. CPU TIME:     0.00 SEC, REAL TIME:     0.00 SEC

 FILE SIZES:   FILE 1:    30.2 MBYTE,  FILE 4:     4.2 MBYTE,   TOTAL:     34.4 MBYTE

 OPERATOR DM      FOR CENTER  0  COORDINATES:    0.000000    0.000000    0.000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       28.54       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

 PROGRAMS   *        TOTAL       INT
 CPU TIMES  *         0.13      0.05
 REAL TIME  *         0.19 SEC
 DISK USED  *        33.05 MB
 **********************************************************************************************************************************


 Program * Restricted Hartree-Fock

 Orbital guess generated from atomic densities.

 Initial occupancy:   1   0   0   0   0

 NELEC=    2   SYM=1   MS2= 0   THRE=1.0D-08   THRD=3.2D-06   THRG=3.2D-06  HFMA2=F  DIIS_START=2   DIIS_MAX=10   DIIS_INCORE=F

 Level shifts:    0.00 (CLOSED)    0.00 (OPEN)    0.30 (GAP_MIN)

 ITER           ETOT              DE          GRAD        DDIFF     DIIS  NEXP   TIME(IT)  TIME(TOT)  DIAG
   1       -2.85516035      -2.85516035     0.00D+00     0.82D+00     0     0       0.00      0.01    start
   2       -2.85516048      -0.00000013     0.37D-03     0.58D-03     1     0       0.00      0.01    diag
   3       -2.85516048      -0.00000000     0.28D-04     0.48D-04     2     0       0.00      0.01    diag
   4       -2.85516048      -0.00000000     0.16D-09     0.15D-09     3     0       0.00      0.01    diag
   5       -2.85516048      -0.00000000     0.73D-10     0.12D-09     0     0       0.00      0.01    diag

 Final occupancy:   1   0   0   0   0

 !RHF STATE 1.1 Energy                 -2.855160477243
  RHF One-electron energy              -3.882025102604
  RHF Two-electron energy               1.026864625362
  RHF Kinetic energy                    2.855176138078
  RHF Nuclear energy                    0.000000000000
  RHF Virial quotient                  -0.999994514932

 !RHF STATE 1.1 Dipole moment           0.00000000     0.00000000     0.00000000
 Dipole moment /Debye                   0.00000000     0.00000000     0.00000000

 Orbital energies:

           1.1          2.1
     -0.914148     1.397442

           1.2
      2.524372

           1.3
      2.524372

           1.5
      2.524372


 HOMO      1.1    -0.914148 =     -24.8752eV
 LUMO      2.1     1.397442 =      38.0263eV
 LUMO-HOMO         2.311590 =      62.9016eV

 Orbitals saved in record  2100.2


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       28.54       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

              2       4        0.34       700     1000      520     2100   
                                         GEOM     BASIS   MCVARS     RHF  

 PROGRAMS   *        TOTAL   RHF-SCF       INT
 CPU TIMES  *         0.14      0.01      0.05
 REAL TIME  *         0.20 SEC
 DISK USED  *        33.05 MB
 **********************************************************************************************************************************


 PROGRAM * MP2 (Closed-shell)     Authors: C. Hampel, H.-J. Werner, 1991, M. Deegan, P.J. Knowles, 1992


 Convergence thresholds:  THRVAR = 1.00D-08  THRDEN = 1.00D-06

 Number of closed-shell orbitals:   1 (   1   0   0   0   0   0   0   0 )
 Number of external orbitals:       4 (   1   1   1   0   1   0   0   0 )

 Molecular orbitals read from record     2100.2  Type=RHF/CANONICAL  

 Number of N-1 electron functions:               1
 Number of N-2 electron functions:               1
 Number of singly external CSFs:                 1
 Number of doubly external CSFs:                 4
 Total number of CSFs:                           6

 Length of J-op  integral file:               0.00 MB
 Length of K-op  integral file:               0.03 MB

 Integral transformation finished. Total CPU:   0.00 sec, npass=  1  Memory used:   0.07 MW

 Memory needed for MP2:                      0.00 MW

 SCS-MP2 correlation energy:           -0.030994007462   (PS=  1.200000  PT=  0.333333)
 SCS-MP2 total energy:                 -2.886154484704


  Reference energy                     -2.855160477243
  MP2 singlet pair energy              -0.025828339551
  MP2 triplet pair energy               0.000000000000
  MP2 correlation energy               -0.025828339551

 !MP2 total energy                     -2.880988816794


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      20       28.54       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700     1380   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER     JKOP   

              2       4        0.34       700     1000      520     2100   
                                         GEOM     BASIS   MCVARS     RHF  

 PROGRAMS   *        TOTAL       MP2   RHF-SCF       INT
 CPU TIMES  *         0.16      0.02      0.01      0.05
 REAL TIME  *         0.22 SEC
 DISK USED  *        33.05 MB
 **********************************************************************************************************************************


 PROGRAM * CCSD (Closed-shell coupled cluster)     Authors: C. Hampel, H.-J. Werner, 1991, M. Deegan, P.J. Knowles, 1992


 Convergence thresholds:  THRVAR = 1.00D-08  THRDEN = 1.00D-06

 Number of closed-shell orbitals:   1 (   1   0   0   0   0   0   0   0 )
 Number of external orbitals:       4 (   1   1   1   0   1   0   0   0 )

 Molecular orbitals read from record     2100.2  Type=RHF/CANONICAL  

 Number of N-1 electron functions:               1
 Number of N-2 electron functions:               1
 Number of singly external CSFs:                 1
 Number of doubly external CSFs:                 4
 Total number of CSFs:                           6

 Length of J-op  integral file:               0.00 MB
 Length of K-op  integral file:               0.03 MB
 Length of 3-ext integral record:             0.00 MB

 Integral transformation finished. Total CPU:   0.00 sec, npass=  1  Memory used:   0.07 MW

 Reference energy:                     -2.85516048

 MP2 singlet pair energy:              -0.02582834
 MP2 triplet pair energy:               0.00000000
 MP2 correlation energy:               -0.02582834
 MP2 total energy:                     -2.88098882

 SCS-MP2 correlation energy:           -0.03099401   (PS=  1.200000  PT=  0.333333)
 SCS-MP2 total energy:                 -2.88615448

 ITER.      SQ.NORM     CORR.ENERGY   TOTAL ENERGY   ENERGY CHANGE        DEN1      VAR(S)    VAR(P)  DIIS     TIME  TIME/IT
   1      1.00658016    -0.03214041    -2.88730089    -0.00631207    -0.00507080  0.32D-06  0.19D-03  1  1     0.00     0.00
   2      1.00728799    -0.03243584    -2.88759632    -0.00029543    -0.00151371  0.62D-07  0.53D-06  2  2     0.00     0.00
   3      1.00730767    -0.03243359    -2.88759407     0.00000225    -0.00002069  0.59D-09  0.59D-12  3  3     0.00     0.00
   4      1.00730820    -0.03243455    -2.88759503    -0.00000096    -0.00000100  0.39D-32  0.66D-35  4  4     0.00     0.00
   5      1.00730810    -0.03243435    -2.88759483     0.00000019     0.00000019  0.57D-42  0.39D-43  5  5     0.00     0.00

 Norm of t1 vector:      0.00110138      S-energy:     0.00000000      T1 diagnostic:  0.00055069
                                                                       D1 diagnostic:  0.00077880
                                                                       D2 diagnostic:  0.08548080 (internal)

 Doubles amplitudes (print threshold =  0.500E-01):

         I         J         SYM. A    SYM. B    A         B      T(IJ, AB)

         1         1         1         1         1         1     -0.06474805


 RESULTS
 =======

  Reference energy                     -2.855160477243
  CCSD singlet pair energy             -0.032434353976
  CCSD triplet pair energy              0.000000000000
  CCSD correlation energy              -0.032434353976

 !CCSD total energy                    -2.887594831219

 Program statistics:

 Available memory in ccsd:                31999994
 Min. memory needed in ccsd:                   138
 Max. memory used in ccsd:                     138
 Max. memory used in cckext:                 32875 ( 5 integral passes)



 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      20       28.54       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700     1380   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER     JKOP   

              2       4        0.34       700     1000      520     2100   
                                         GEOM     BASIS   MCVARS     RHF  

 PROGRAMS   *        TOTAL      CCSD       MP2   RHF-SCF       INT
 CPU TIMES  *         0.18      0.02      0.02      0.01      0.05
 REAL TIME  *         0.25 SEC
 DISK USED  *        33.05 MB
 **********************************************************************************************************************************

 CCSD/cc-pVDZ energy=     -2.887594831219

            CCSD             MP2         RHF-SCF
     -2.88759483     -2.88098882     -2.85516048
 **********************************************************************************************************************************
 Molpro calculation terminated
